projects/1.18-fabric/assets/packages/packages/lang/zh_cn.json

```
@@ -0,0 +1,62 @@
+{
+	"itemGroup.packages.group": "包裹",
+
+	"block.packages.package_maker": "打包台",
+	"block.packages.package": "package",
+	"block.packages.package.nonempty": "%s (%s)",
+	"block.packages.package.nonempty.reentrant": "(%s)",
+	"block.packages.package.nonempty.contents": "%sx %s",
+
+	"packages.style_tooltip.hold_for_composition": "按住[%s]查看详情",
+	"packages.style_tooltip.shift": "Shift",
+	"packages.style_tooltip.frame": "%s 框架",
+	"packages.style_tooltip.inner": "%s 内衬",
+	"packages.style_tooltip.both": "%s 框架和内衬",
+	"packages.style_tooltip.color": "%s 正面",
+	"packages.style_tooltip.color.white": "白色",
+	"packages.style_tooltip.color.orange": "橙色",
+	"packages.style_tooltip.color.magenta": "品红色",
+	"packages.style_tooltip.color.light_blue": "淡蓝色",
+	"packages.style_tooltip.color.yellow": "黄色",
+	"packages.style_tooltip.color.lime": "黄绿色",
+	"packages.style_tooltip.color.pink": "粉红色",
+	"packages.style_tooltip.color.gray": "灰色",
+	"packages.style_tooltip.color.light_gray": "淡灰色",
+	"packages.style_tooltip.color.cyan": "青色",
+	"packages.style_tooltip.color.purple": "紫色",
+	"packages.style_tooltip.color.blue": "蓝色",
+	"packages.style_tooltip.color.brown": "棕色",
+	"packages.style_tooltip.color.green": "绿色",
+	"packages.style_tooltip.color.red": "红色",
+	"packages.style_tooltip.color.black": "黑色",
+
+	"packages.contents_tooltip.utimately": "-> %s",
+
+	"packages.zzzzzzzzz.hello.1": "（编辑本文件的人注意: 以下tooltip键的行数是动态的）",
+	"packages.zzzzzzzzz.hello.2": "（如果你想添加更多行，只需增加更多的语言键）",
+	"packages.zzzzzzzzz.hello.3": "（如果您想缩短字符串ZZZ的长度，将语言键设置为字符串ZZZ将使其充当列表结束信号）",
```

对应的代码在[这里](https://github.com/quat1024/packages/blob/11c84284826259774315ecf3d0d370873677cf1f/Xplat/src/main/java/agency/highlysuspect/packages/client/PackageMakerScreen.java#L77-L83)：
```java
				for(int k = 1; true; k++) {
					String line = tooltip + "." + k;
					if(!Language.getInstance().has(line)) break;
					//this lets you make the tooltip have fewer lines using a resourcepack (which can't actually remove lang keys, only override them)
					if(Language.getInstance().getOrDefault(line).contains("ZZZ")) break;
					toot.add(ILoveMojang.translatable(line).withStyle(ChatFormatting.DARK_GRAY));
				}
```
那么，这三行的用途有可能相当于注释
当然，注释也不是不能翻译，只不过：
```suggestion
	"packages.zzzzzzzzz.hello.3": "（如果要截断某些文本，在下一个键的文本中包含字符串ZZZ）",
```

---

对，这一段我确实没太看得懂，也没看懂这个字符串的用法，于是凭感觉翻译了一次，好在游戏里并不会出现它这三行。